ros/ros2/OdometryServer.hpp

```diff
@@ -37,7 +37,7 @@ namespace kiss_icp_ros {
 class OdometryServer : public rclcpp::Node {
 public:
     /// OdometryServer constructor
-    OdometryServer();
```

Comment:
Should this be deleted? Otherwise, the compiler will generate the default constructor. Or does this even matter for the component's API?

Reply:
For components API, the only requirement is to have the constructor taking node options. You mean explicitly `= delete;` the default one?

Reply:
Ah, another requirements to leverage true zero-copy is to use ConstSharedPtr instead of SharedPtr of ROS2 messages in the callback signature. I'll push the patch now.

Reply:
Also, I checked but the default constructor is automatically "deleted" since there is an user-defined constructor (https://en.cppreference.com/w/cpp/language/default_constructor)